Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -13,13 +13,43 @@ function connected(jsn) {
  *       url?: string,
  *       method?: string,
  *       body?: string|null,
+ *       contentType?: string,
+ *       headers?: string[]|null,
+ *       body?: string|null,
  *     }
  *   },
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = {
+        'Content-Type': typeof contentType !== 'undefined' && contentType ? contentType : 'text/plain'
+    };
+    let inputHeaders = {};
+
+    if (headers) {
+        const headersArray = headers.split(/\n/);
+
+        for (let i = 0; i < headersArray.length; i += 1) {
+            if (headersArray[i].includes(':')) {
+                const [headerItem, headerItemValue] = headersArray[i].split(':');
+                const trimmedHeaderItem = headerItem.trim();
+                const trimmedHeaderItemValue = headerItemValue.trim();
+
+                inputHeaders[trimmedHeaderItem] = trimmedHeaderItemValue;
```

Comment:
Having an empty string as a header name will still cause an error. We need to check for that.